docs/interactive/comments/dokieli.md

```diff
@@ -0,0 +1,25 @@
+# Dokieli
+
+```{raw} html
+<script src="https://dokie.li/scripts/dokieli.js"></script>
+<link media="all" rel="stylesheet" type="text/css" href="https://dokie.li/media/css/dokieli.css" />
```

Comment:
<link href="https://dokie.li/media/css/dokieli.css" media="all" rel="stylesheet" />
    <script src="https://dokie.li/scripts/dokieli.js"></script>
```suggestion
<script src="https://dokie.li/scripts/dokieli.js"></script>
<link media="all" rel="stylesheet" type="text/css" href="https://dokie.li/media/css/dokieli.css" />
```

Reply:
I believe it's because `require.js` is loaded in the netlify preview page, but it is not loaded in the `sphinx-comments` page. That's why I thought this was a require.js clash

Reply:
I'm not familiar with the other scripts. Is require.js a netfly-specific thing? In case it makes any differences, I'd suggest to include the CSS and JS in `<head>` any way. BTW, the script listens to DOMContentLoaded.

Reply:
hmm maybe it's not requirejs? I just am seeing this error on the dokieli page on the jupyter book preview:

![image](https://user-images.githubusercontent.com/1839645/89928954-68b1b500-dbbd-11ea-979b-8b6be5d56b34.png)

re: putting in head, in this case I'm just putting it directly into the body of the page because I *only* want to activate it on that page but not the others and this was the only way I could figure out how to make it happen (since this is being built with Sphinx we don't have easy page-specific control).

Reply:
Experimenting a little bit more, I found that dokieli *sometimes* works and *sometimes* doesn't on the `sphinx-comments` site when it is placed in the `head`. For example, I tried refreshing this page a bunch of times:

https://sphinx-comments--11.org.readthedocs.build/en/11/index.html

Maybe 1 out of 8 times dokieli didn't load (with no error message). So it seems there is some kind of race condition it is hitting? I can't find any kind of "best practices for installing and activating dokieli" documentation, is there something like this?